action.yml

```diff
@@ -28,7 +28,7 @@ inputs:
     required: false
   lycheeVersion:
     description: "Use custom version of lychee link checker"
-    default: v0.15.0
+    default: v0.16.1
```

Comment:
Ah, the version syntax should actually stay the same; only the release name (git tag) was affected. At least the pipeline is green -- and that uses the default -- so that's why I assume it's correct. 😆